src/content/extensions/multipleroutes/src/_locales/en/messages.json

```diff
@@ -1,15 +1,18 @@
 {
-   "NETLI_MULTIROUTEONUDPON_RADIO": {
+   "NETLI_DEFAULT_RADIO": {
```

Comment:
Should we call this something without the word "default" in it, like "NETLI_BEST_MEDIA_RADIO"?